### DISCUSSION TITLE
⌛ 🔗 Show/Hide password in login form as it is the default WordPress behavior

### DISCUSSION THREAD

**Discussion by amikeliunas** [elementor/elementor#21243] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The Elementor Pro LOGIN widget does not show/hide password in login form as it is the default WordPress behavior.

### Describe the solution you'd like

What I would like is:
 1) a fix for the Pro login widget [on/off toggle switch] 

### Describe alternatives you've considered

 2) a snippet or custom Elementor code based on this https://stackoverflow.com/questions/69908009/placing-password-visibility-icon-inside-password-field 

### Additional context

to make the Elementor Pro Login widget behave like the DEFAULT WordPress password field show or hide password text.

**Comment by amikeliunas** [comment]
See Elementor Pro Login widget [fails to show password] here https://www.ajbid.org/en/member-login/
versus default WordPress login form with EYE ICON to show or hide password content here
https://www.ajbid.org/wp/wp-login.php

**Comment by nicholaszein** [comment]
This Request was mentioned in the following Issues and Discussions:

- #21264

**Comment by drewjohnstone** [comment]
Is there any update on this?
